app/src/androidTest/java/com/github/epfl/meili/models/ChatMessageUnitTest.kt

```diff
@@ -1,41 +0,0 @@
-package com.github.epfl.meili.models
```

Comment:
Why is this file deleted? It should be preserved right?

Reply:
I have no idea how this happened. I know I deleted tests to see if mine passed, but I thought I had restored them. I will put them back in.